### PR TITLE
UI View changes for Unit -> Manage Roles

### LINKED ISSUE #7883: Update Unit assign roles view to new wireframe
Need wait for Michał and Bill discuss conclusions

### PR DESCRIPTION
**Pull request by lohithosur**
Hi Michal,
So far:
 - I have adopted UI suggestions as per Bill's wireframe mockup.
 - The relevant columns (4 checkboxes) are shown as checked/unchecked depending on the value it contains.

Need to work on:
 - Handling of check/uncheck of 4 checkboxes -> need to assign roles on check/uncheck of checkboxes? May be trigger relevant event(?). 

Any idea how to achieve this? I'm only finding examples of checkboxes as a part of form. I could not find how to achieve it in the context of interactive table column.

Regards,
Lohit

### CODE REVIEW

File: projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.ts [hunk 1]
```diff
@@ -66,30 +67,55 @@ export class UnitAssignRolesComponent extends AbstractListingComponent
               parentUnit: user.orgUnit && user.orgUnit.name,
               uid: user.orgUnit && user.orgUnit.uid,
               customerId: user.customerId,
+              admin: user.roles.includes('b2badmingroup'),
+              approver: user.roles.includes('b2bapprovergroup'),
+              customer: user.roles.includes('b2bcustomergroup'),
+              manager: user.roles.includes('b2bmanagergroup'),
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.ts]: what do you think about
```suggestion
              b2badmingroup: user.roles.includes('b2badmingroup'),
              b2bapprovergroup: user.roles.includes('b2bapprovergroup'),
              b2bcustomergroup: user.roles.includes('b2bcustomergroup'),
              b2bmanagergroup: user.roles.includes('b2bmanagergroup'),
```
In that case we should also change translation keys and then we can remove `getRole` method

File: projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.spec.ts [hunk 2]
```diff
@@ -221,21 +235,15 @@ describe('UnitAssignRolesComponent', () => {
 
   describe('assign', () => {
     it('should assign user', () => {
-      component.assign(userRow);
-      expect(orgUnitService.assignRole).toHaveBeenCalledWith(
-        userRow.row.customerId,
-        roleId
-      );
+      component.assign(event);
+      expect(b2bUsersService.update).toHaveBeenCalled();
```
Comment (Pucek9) [inline projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.spec.ts]: https://sap-cx.slack.com/archives/C0118U91T54/p1588940533003100

**Review by Pucek9 — CHANGES_REQUESTED** [review]
Please follow comments

File: projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.ts (hunk 1 above)
Comment (lohithosur) [reply projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.ts]: Done

File: projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.spec.ts (hunk 2 above)
Comment (lohithosur) [reply projects/storefrontlib/src/cms-components/organization/units/unit-assign-roles/unit-assign-roles.component.spec.ts]: Done

**Review by Pucek9 — APPROVED** [review]

**Pull request merged**